tests/D2L.CodeStyle.Analyzers.Test/Specs/DependencyRegistrationsAnalyzer.cs

```diff
@@ -229,10 +241,16 @@ public interface IImmutableSingleton : ISingleton { }
 	public interface IImmutableSubSingleton : IImmutableSingleton { }
 
 	[Objects.Immutable]
-	public sealed class ImmutableThing : ISingleton, INotSingleton { }
+	public sealed class ImmutableThing : IMarkedSingleton, ISingleton, INotSingleton { }

```

Comment:
Can you undo this change?  `ISingleton` and `IMarkedSingleton` are both empty interfaces with the `[Singleton]` attribute.  Seems like we only need one of them